Marlin/ultralcd.cpp

```diff
@@ -1543,6 +1547,13 @@ void lcd_init() {
       WRITE(SHIFT_LD, HIGH);
     #endif
 
+    #ifdef RIGIDBOT_PANEL
+      pinMode(BTN_UP,INPUT);
+      pinMode(BTN_DWN,INPUT);
+      pinMode(BTN_LFT,INPUT);
+      pinMode(BTN_RT,INPUT);
+    #endif
+
```

Comment:
Maybe `SET_INPUT` is preferable before `pinMode`. We already have enough problems where Arduino an FastIO pin mapping is confused.